src/output/icons.rs

```diff
@@ -405,12 +457,15 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "dll"            => Icons::LIBRARY,          // 
     "dmg"            => Icons::DISK_IMAGE,       // 
     "doc"            => Icons::DOCUMENT,         // 
+    "docm"           => Icons::DOCUMENT,         // 
     "docx"           => Icons::DOCUMENT,         // 
-    "dot"            => '\u{f1049}',             // 󱁉
+    "dot"            => Icons::GRAPH,            // 󱁉
```

Comment:
This is not related to nerdfonts 3.1.0 either

Reply:
That's why the PR says "** & more** at the end :wink: